content/docs/1.1.0/references/longhorn-client-python.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Python Client
+weight: 5
+---
+
+Currently, you can operate Longhorn using Longhorn UI.
+We are planning to build a dedicated Longhorn CLI in the upcoming releases.
+
+In the meantime, you can access Longhorn API using Python binding, as we demonstrated below.
+
+1. Get Longhorn endpoint
+
+   One way to communicate with Longhorn is through `longhorn-frontend` service.
+   If you run your automation/scripting tool inside the same cluster in which Longhorn is installed, connect to the endpoint `http://longhorn-frontend.longhorn-system/v1`
+
+   If you run your automation/scripting tool on your local machine,
+   use `kubectl port-forward` to forward the `longhorn-frontend` service to localhost:
+   ```
+   kubectl port-forward services/longhorn-frontend 8080:http -n longhorn-system
+   ```
+   and connect to endpoint `http://localhost:8080/v1`
+
+2. Using Python Client
+
+    Import file [longhorn.py](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/longhorn.py) which contains the Python client into your Python script and create a client from `longhorn_url`
```

Comment:
`from the endpoint` since there is no mention of `longhorn_url` before.

Reply:
Thank you. Fixed